packages/core/integration-tests/test/schema-jsonld.js

```diff
@@ -1,26 +1,88 @@
-import {bundle, assertBundleTree} from '@parcel/test-utils';
+import {bundle, assertBundles, distDir, outputFS} from '@parcel/test-utils';
+import assert from 'assert';
+import path from 'path';
 
-describe.skip('schema ld+json', function() {
+function getPathToFile(relativePathToFile) {
+  return path.join(__dirname, relativePathToFile);
+}
+
+let pathToIndex = getPathToFile('/integration/schema-jsonld/index.html');
+
+async function getBundleFile() {
+  await bundle(pathToIndex);
+  let pathToOutputFile = path.join(distDir, 'index.html');
+  return outputFS.readFile(pathToOutputFile, 'utf-8');
+}
+
+describe('jsonld', function() {
   it('Should parse a LD+JSON schema and collect dependencies', async function() {
-    let b = await bundle(__dirname + '/integration/schema-jsonld/index.html', {
+    let b = await bundle(pathToIndex, {
       production: true,
       publicURL: 'https://place.holder/',
     });
 
-    await assertBundleTree(b, {
-      name: 'index.html',
-      assets: ['index.html'],
-      childBundles: [
-        {
-          type: 'jpeg',
-        },
-        {
-          type: 'png',
-        },
-        {
-          type: 'css',
-        },
-      ],
-    });
+    assertBundles(b, [
+      {
+        name: 'index.html',
+        assets: ['index.html'],
+      },
+      {
+        type: 'css',
+        assets: ['other.css'],
+      },
+      {
+        type: 'jsonld', //this is the jsonld asset
+        assets: ['index.html'],
+      },
+      {
+        type: 'png',
+        assets: ['logo.png'],
+      },
+      {
+        type: 'jpeg',
+        assets: ['image.jpeg'],
+      },
+    ]);
+  });
+
+  it('Should output the original json back into the index.html file inside the script tag', async function() {
+    /* let v1JSONLDOutputInsideScriptTag = {
+      '@context': 'http://schema.org',
+      '@type': 'LocalBusiness',
+      description: 'This is your business description.',
+      name: "Parcel's parcel",
+      telephone: '555-111-2345',
+      openingHours: 'Mo,Tu,We,Th,Fr 09:00-17:00',
+      logo: {
+        '@type': 'ImageObject',
+        url: 'images/logo.png',
+        width: 180,
+        height: 120,
+      },
+      image: ['image\\.[a-f0-9]+/\\.jpeg', 'images/image.jpeg'],
+    }; */
+
+    let file = await getBundleFile();
```

Comment:
As I said already, there is no value in having two tests for this:
- the second test shouldn't read the output of the first test (as it does now). I'm actually surprised that this works as at all because the output filesystem is cleared before every test..
- if the first test fails, the second one will fail as well (if the dependencies aren't collected).

Reply:
that test's intention was to make sure that the content being inserted inbetween the script tags was actually a json object rather than the 'div' tag that was previously appearing there. Since then, it has changed and become redundant, as you pointed out. I will remove the test. 

However, if you think the original intention of the test is a worthy pursuit, (make sure a json object is inserted in between the script tag) I can write that. The true test for that would be to extract the content inbetween the script tag, like its currently doing, but then just test that the content is a valid json object instead of looking at the content of its properties / assets.

Reply:
Sorry, I should have written that in the beginning: I meant that you should merge these two tests, so:
```js
it('...', async () => {
	let b = bundle();
	
	assertBundles(b, {
	...
	});

    let regex = new RegExp(
      /<\s*script \s*type="application\/ld\+json"\s*>.*<\/\s*script\s*>/gm,
    );
    let firstMatch = regex.exec(file).map(m => {
      // removes script html tags to extract the json object in between them
      return m.replace(/<\/?\s*script( \w+=".*"\s*)*\s*>/g, '');
    })[0];

    let actual = JSON.parse(firstMatch);
    match(actual.logo.url, /logo\.[a-f0-9]+\.png/);
    match(actual.image[0], /image\.[a-f0-9]+\.jpeg/);
    match(actual.image[1], /image\.[a-f0-9]+\.jpeg/);
});
```
(Not really important, but you can simplify that regex logic to `let firstMatch = file.match(/...../gm)[1]` with the regex `<\s*script \s*type="application\/ld\+json"\s*>(.*)<\/\s*script\s*>` (exactly what you wrote but using a capture group for the script tag contents))

Reply:
see 3e6f25904390c5a1ebda60e69c135c5f6be47495